app/components/base.scss

```diff
@@ -1,4 +1,9 @@
 @import "../webpacker/styles/govuk_style_setup";
 @import "~govuk-frontend/govuk/base";
 
-$moj-images-path: "~@ministryofjustice/frontend/moj/assets/images/";
\ No newline at end of file
+$moj-images-path: "~@ministryofjustice/frontend/moj/assets/images/";
+
+$mobile-small-start: 400px;
+$mobile-small-end: 401px;
+$mobile-big-start: 500px;
+$mobile-big-end: 501px;
```

Comment:
I might look at how the design system does breakpoints - my hunch is we just need one number per breakpoint, not two.